narwhals/_pandas_like/series.py

```diff
@@ -486,6 +486,12 @@ def zip_with(self: Self, mask: Any, other: Any) -> PandasSeries:
         res = ser.where(mask._series, other._series)
         return self._from_series(res)
 
+    def head(self: Self, n: int = 10) -> Self:
```

Comment:
general point but we should probably avoid specifying defaults in `_pandas_like`, so that defaults (if any) get passed down from above